app/scripts/lib/open-update-tab-and-reload.test.ts

```diff
@@ -0,0 +1,46 @@
+import browser from 'webextension-polyfill';
+import { openUpdateTabAndReload } from './open-update-tab-and-reload';
+
+jest.mock('webextension-polyfill', () => ({
+  tabs: {
+    create: jest.fn(),
+  },
+}));
+
+describe('openUpdateTabAndReload', () => {
+  const mockRequestSafeReload = jest.fn();
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+  afterEach(() => {
+    jest.restoreAllMocks();
+  });
```

Comment:
Nit: This happens automatically after each test already, so we shouldn't need this (see the `restoreMocks` option in our jest config)

```suggestion
```